lib/knackhq/client.rb

```diff
@@ -70,8 +54,8 @@ def update_record(object, knackhq_id, json)
                      .objects(object)
                      .records(knackhq_id)
                      .put(:body => json)
-                     .to_h
-      !hash_request.empty?
+      payload = payload_hash(hash_request)
+      translate_payload(payload) { payload }
```

Comment:
Yes. This behavior made it easy to test. When we had the depreciated Blanket gem, the output was always the same, so we could never test to see if the object has changed, which led to the issue at hand. Ill change this behavior to return true/false as we discussed before.